delivery_roulier/models/stock_pack_operation.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api
+from odoo.tools import float_is_zero
+
+
+class StockPackOperation(models.Model):
+    _inherit = 'stock.pack.operation'
+
+    @api.multi
+    def _roulier_get_unit_price_for_customs(self):
+        """This method is designed to be inherited for specific scenarios"""
```

Comment:
If this method is not dependent of the carrier, it should nammed wihtout _roulier prefix.